src/components/ChatLog.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.jsx';
+import './ChatLog.css';
+
+const ChatLog = ({ entries, onToggleLike }) => {
+  const chatEntryComponents = entries.map((entry) => {
+    return (
+      <ChatEntry
+        id={entry.id}
+        sender={entry.sender}
+        body={entry.body}
+        timeStamp={entry.timeStamp}
+        liked={entry.liked}
```

Comment:
Since the names of the keys on `entry` are the same as the names your are setting on `ChatEntry` attributes/you are using all of the values in `entry` you could do something something like this to save you a few keystrokes: 

```javascript
  const chatComponents = entries.map((entry) => {
    return(
      <ChatEntry
        {...entry}
        onLikeToggle={onLikeBtnToggle}
        key={entry.id}
      />
    );
  });
```
Just be mindful that this isn't as explicit as what you have.